views/project/user.erb

```diff
@@ -83,7 +83,7 @@
           </tr>
         <% end %>
         <% @invitations.each do |invitation| %>
-          <tr id="invitation-<%= invitation[:email] %>" class="whitespace-nowrap text-sm font-medium">
+          <tr id="invitation-<%= invitation[:email].gsub(/\W+/, "") %>" class="whitespace-nowrap text-sm font-medium">
```

Comment:
What is the purpose of `gsub` here?

Reply:
When we use `@` or `.` in the `id` attribute, it conflicts with CSS selectors.